.github/workflows/ci.yaml

```diff
@@ -63,21 +63,52 @@ jobs:
 
   test-musl:
     runs-on: ubuntu-latest
-    container: clux/muslrust:stable
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
 
-      - name: Link to predefined musl toolchain
+      - name: Build binary
```

Comment:
At one point I needed to change this to work with Docker directly for submodules to work, but since that is no longer necessary these changes can be reverted.